recipe/meta.yaml

```
@@ -1,44 +1,50 @@
-{% set version = "1.10.0" %}
+{% set name = "libssh2" %}
+{% set version = "1.11.0" %}
 
 package:
-  name: libssh2
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://www.libssh2.org/download/libssh2-{{ version }}.tar.gz
-  sha256: 2d64e90f3ded394b91d3a2e774ca203a4179f69aebee03003e5a6fa621e41d51
+  url: https://www.libssh2.org/download/{{ name }}-{{ version }}.tar.gz
+  sha256: 3736161e41e2693324deb38c26cfdc3efe6209d634ba4258db1cecff6a5ad461
 
 build:
-  number: 3
+  number: 0
   run_exports:
     - {{ pin_subpackage('libssh2') }}
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - make   # [unix]
     # This breaks a dependency cycle:
     # curl->libssh2->cmake->curl
     - cmake-no-system
+    - ninja-base  # [win]
+    - make        # [unix]
```

```suggestion
    - ninja-base
```
We can use Ninja on both platforms.

---

Done